lib/smart_answer/question/checkbox.rb

```diff
@@ -38,7 +38,9 @@ def parse_input(raw_input)
       end
 
       def to_response(input)
-        input.split(",").reject { |v| v == NONE_OPTION }
+        input.split(",")
+      rescue StandardError
+        nil
```

Comment:
Rather than use standard error, that might conceal other exceptions should we do `input.split(",") if input.respond_to?(:split)`

Reply:
Thanks! I've fixed and fixed-up.